add_user.php

```diff
@@ -28,47 +28,60 @@
 			<!-- import the webpage's javascript -->
 			<!-- nav javascript -->
 			<script src="js/nav.js" defer></script>
+			<!-- show password javascript -->
+			<script src="js/showpword.js" defer></script>
+			<!-- back to top button javascript -->
+			<script src="js/backtotop.js" defer></script>
 			</head>
 	
 	
         <body>
-            <!-- navigation menu (burger menu)-->
+            <!-- navigation menu-->
 			<?php
 			  require("nav.php");
 			?>
 			
-                                <div class="content"><!-- holds the main content on the admin pages (different styling to other pages) -->
-									
-										<h3><form method="post" id="add_user.php">
-											<label style="color: black;" for = 'login'>Username:</label>
-											<input type = "text" name= "username" placeholder = "Enter Username"/><br>
-											<label style="color: black;" for = 'login'>Password:</label>
-											<input type = "password" name= "password" placeholder = "Enter Password"/><br>
-											<input type = "submit" value = "Insert"/><br>
-										</form></h3>
-											
-										<?php
+                                <div class="admin-content"><!-- Holds the main page content -->
```

Comment:
#used different styling for main content on admin pages as they needed to be smaller than home and query pages